code/modules/client/preference_setup/loadout/loadout_mask.dm

```diff
@@ -82,3 +82,7 @@
 
 /datum/gear/mask/lace/New()
 	gear_tweaks += gear_tweak_free_color_choice
+
```

Comment:
Please //ChompEDIT START ... //ChompEDIT END block comment code in upstream touched files, or put edits in modular_chomp code files instead